packages/web/src/scss/components/TextField/_TextField.scss

```diff
@@ -176,6 +190,13 @@ $_field-name: 'TextField';
     cursor: cursors.$disabled;
 }
 
+// 1.
```

Comment:
Missing `stylelint-disable`?

Reply:
it is folded, but there is another stylelint-disable above